### PR TITLE
Add single line comment in zenoh-examples

### PR DESCRIPTION
**Pull request by sreeja**

### CODE REVIEW

File: examples/src/lib.rs [hunk 1]
```diff
@@ -1 +1 @@
-
+//! Examples on using Zenoh. Check the Readme for usage.
```
Comment (JEnoch) [inline examples/src/lib.rs:1]: For even more clarity I would reword as such:
```rust
//! Examples on using Zenoh. 
//! See the code in ../examples/ 
//! Check ../README.md for usage.
```

Comment (sreeja) [reply examples/src/lib.rs:1]: I agree.

**Review by JEnoch — APPROVED** [review]
